engwales-opinion.sty

```diff
@@ -0,0 +1,52 @@
+\ProvidesPackage{engwales-opinion}[2021-02-13 Package for pleading creation in England/Wales styles]
+
+\RequirePackage[explicit]{titlesec}
+\RequirePackage{enumitem}
+
+
+%% Formatting the main title
+\renewcommand\maketitle{%
+  \vspace*{-1.5cm}
```

Comment:
users should be responsible for setting a correct margin, not `\maketitle`. if there are regulations on the margins, we can later convert this to a .cls file (maybe we already should).

Reply:
I will add this to the discussion tab, but just so that you have this: https://www.civillitigationbrief.com/2017/11/05/fonts-lawyers-and-the-rules-never-ever-use-comic-sans/

This is a blog post that covers the relevant Practice Directions in England and Wales. Unlike the US, specific margins are not required.

Reply:
> if there are regulations on the margins, we can later convert this to a .cls file (maybe we already should).

Yea, even if there are no regulations, a .cls file would probably make more sense, as this package won't be used within any other class anyway.

Reply:
I could make it auto-scale to chosen margins, then if a option is passed use that option. Seems the most get out of your way solution, while still allowing customisation.

Reply:
And in regards to the .cls point, it might be best to just pass arguments to it to choose which layouts are used. Creating a multi-purpose (legal) class with several layout options. Though that's out of the scope of this PR, might be a good idea to implement.

Reply:
>I could make it auto-scale to chosen margins, then if a option is passed use that option. Seems the most get out of your way solution, while still allowing customisation.

Is this idea to announce normal margins so that it's easier to go back and make them wider, if need be?

Reply:
>Creating a multi-purpose (legal) class with several layout options. Though that's out of the scope of this PR, might be a good idea to implement.

How can we start this? I like this idea!!

Reply:
It would probably be best to either work on it after the opinion and pleading .sty s are to a functioning level, or to start working on the framework now, and then start adding features again. You might want to create either an issue with the feature tag, or a discussion to talk about it as it is more general project direction. Also if that's the road that is decided on, it might be best to work out some agreed practices before work commences.

Reply:
That's a great idea.

I think it would be easier if there was a live chat that we could work on, so that I can feed you and @yjyao and @Giebisch all the information you need, and I can answer all the questions that are required. Do you think we can use slack or something similar?

Once we have a live chat so that I get a better idea of the scope, I can give you all more solid answers as to what you need to move forward on this. But if a live chat is not needed, then the structure of a discussion would be good too

Reply:
> > I could make it auto-scale to chosen margins, then if a option is passed use that option. Seems the most get out of your way solution, while still allowing customisation.
> 
> Is this idea to announce normal margins so that it's easier to go back and make them wider, if need be?

Essentially. A default margin would be decided, then by passing arguments to the class (or package or geometry [most likely the messiest to make, but best to implement]) the margins would change with other things on the page adapting to keep pleasing proportions. I'm hesitant to include a practice for doing this within the opinion.sty as a unified approach within a class would probably be best to perform it. Though it wouldn't be too much issue to get it working here if that's what is wanted.

Reply:
>as a unified approach within a class would probably be best to perform it

I am all for it. If a unified approach makes things better from a coding perspective, we should do that.

Reply:
> Communication channel

We could, though currently on 4 people seem active on this so I hesitate about setting up a rigorous system yet. Slack, discord, discourse, IRC, etc. are all options, though we could probably manage with the discussion tab in GitHub for now. Unless you were thinking about a live call, in which case discord would be your best bet for ease of set up IMO. Though probably best to get everyone else's two cents on the matter first.

Reply:
I don't want to eliminate the possibility of a live call just yet.

Let's use the discussion forum for now, and schedule a call in the coming week, whenever we feel that we need to come back into focus on something. I'm sure by that point we will be able to get everyone on board for a call, and I will be able to get another lawyer or two involved, so that we can progress with universalising what we have.

Sorry for the stupid question, but to improve buy-in, is there a specific title you would suggest for this discussion forum thread? Thanks so much!
I'm really enjoying working with you on this

Reply:
> discussion title

Sorry, can't really think of anything. I think something like 'Overview' or 'Ideas' should work. There's probably a best practice for it, but no idea what it is. 'Architecture' might be it (?), that's what the document documenting how everything interacts and where it all goes is generally called. But anything sensible should work.

> call

That's probably be good to get a good start.

> more people

Always a good thing. Their uses and opinions will probably help with creating something widely applicable (to lawyers practising in England or Wales).

Reply:
>Always a good thing. Their uses and opinions will probably help with creating something widely applicable (to lawyers practising in England or Wales).

Exactly!!

>Sorry, can't really think of anything. I think something like 'Overview' or 'Ideas' should work. There's probably a best practice for it, but no idea what it is. 'Architecture' might be it (?), that's what the document documenting how everything interacts and where it all goes is generally called. But anything sensible should work.

Perfect!


>>call
>That's probably be good to get a good start.

Let me get this sorted then!!

Reply:
this thread got way out of the topic lol. so i guess we can just remove the `\vspace` in this PR and leave margin control in upcoming PRs. @bond7o6 sound good?

Reply:
yea, sounds solid.